redfish-core/include/redfish.hpp

```diff
@@ -84,6 +84,11 @@ class RedfishService
 #endif
         requestRoutesThermalSubsystem(app);
         requestRoutesThermalMetrics(app);
+#ifdef BMCWEB_NEW_POWERSUBSYSTEM_THERMALSUBSYSTEM
+        requestRoutesPowerSubsystem(app);
+        requestRoutesPowerSupplyCollection(app);
+        requestRoutesPowerSupply(app);
```

Comment:
Look at line 99 - 101, these are already here. 
I am fine if you move to the ifdef but then need to remove the old ones 

If you want 

 #ifdef BMCWEB_NEW_POWERSUBSYSTEM_THERMALSUBSYSTEM
         requestRoutesThermalSubsystem(app);
        requestRoutesThermalMetrics(app);
         requestRoutesPowerSubsystem(app);
        requestRoutesPowerSupplyCollection(app);
        requestRoutesPowerSupply(app);
  #endif
  
  or just leave this file alone